.github/workflows/rolling.yaml

```diff
@@ -30,15 +32,15 @@ jobs:
         with:
           path: ros_ws/src
       - name: Setup ROS environment
-        uses: ros-tooling/setup-ros@0.3.4
+        uses: ros-tooling/setup-ros@0.7.9
         with:
           required-ros-distributions: ${{ matrix.ros_distribution }}
       - name: Install requirements
         run: pip3 install requests
       - name: Build and Test
-        uses: ros-tooling/action-ros-ci@0.2.6
+        uses: ros-tooling/action-ros-ci@0.3.15
         with:
           package-name: qualisys_driver
           target-ros2-distro: ${{ matrix.ros_distribution }}
-          vcs-repo-file-url: "https://raw.githubusercontent.com/MOCAP4ROS2-Project/mocap4ros2_qualisys/main/dependency_repos.repos"
+          vcs-repo-file-url: "https://raw.githubusercontent.com/MOCAP4ROS2-Project/mocap4ros2_qualisys/rep-mocap/dependency_repos.repos"
```

Comment:
Please change this to ${GITHUB_WORKSPACE}/dependency_repos.repos

Reply:
Hello! Thank you very much ☺️. The issue is that I tried using that variable, but it didn’t pass the CI. I’ll give it another try.

Reply:
Done